### PR TITLE
Update akash commands

### PR DESCRIPTION
**Pull request by akhilkumarpilli**

### CODE REVIEW

File: backend/faucet.go [hunk 1]
```diff
@@ -140,8 +145,17 @@ func CheckAccountBalance(address string, amountFaucet string, key string) error
...
-		accErr != nil {
+	if len(queryRes.Value.Coins) != 0 {
+		f, err := strconv.ParseFloat(queryRes.Value.Coins[0].Amount, 64)
+		if err != nil {
+			return nil
+		}
+		balance = f
+	} else {
+		return nil
+	}
```
Comment (anilcse) [inline backend/faucet.go]:
```suggestion
	if len(queryRes.Value.Coins) == 0 {
		return nil
	}
	
	balance, err := strconv.ParseFloat(queryRes.Value.Coins[0].Amount, 64)
	if err != nil {
		return nil
	}
```

File: backend/faucet.go [hunk 2]
```diff
@@ -140,8 +145,17 @@ func CheckAccountBalance(address string, amountFaucet string, key string) error
...
+
+	if balance < float64(10)*math.Pow(10, 6) || accErr != nil {
```
Comment (anilcse) [inline backend/faucet.go]: Use MAX_TOKENS_ALLOWED config variable. 
```suggestion
	if balance < MAX_TOKENS_ALLOWED || accErr != nil {
```

File: backend/faucet.go [hunk 3]
```diff
@@ -167,7 +181,7 @@ func getCoinsHandler(res http.ResponseWriter, request *http.Request) {
 	// 	panic(encodeErr)
 	// }
 
-	if len(address) != 40 {
+	if len(address) != 44 {
```
Comment (anilcse) [inline backend/faucet.go]: Use a config/constant or use regex

**Review by anilcse — CHANGES_REQUESTED** [review]
Looks good, just a few changes

**Review by anilcse — APPROVED** [review]
lgtm, can you just change float64 to int64 for balances?

**Comment by akhilkumarpilli** [comment]
I think float64 is fine. Balance will be in decimals when we withdraw rewards.

**Comment by anilcse** [comment]
No, chain doesn't support decimals. It's just the display on the client side that will display decimals. Cosmos balances are all i64

**Comment by akhilkumarpilli** [comment]
But we are executing client commands for checking balance. Will it be fine if we change to i64?

**Comment by anilcse** [comment]
Understand, no problem. Let's keep it f64

**Pull request merged**